pkg/printers/printer.go

```diff
@@ -128,7 +128,7 @@ func (c *Printer) createPrinter(format string, w io.Writer) (issuePrinter, error
 	case config.OutFormatHTML:
 		p = NewHTML(w)
 	case config.OutFormatJunitXML, config.OutFormatJunitXMLExtended:
```

Comment:
Do we need to rename these config values as well?
```suggestion
	case config.OutFormatJUnitXML, config.OutFormatJUnitXMLExtended:
```